ci-conda.Dockerfile

```diff
@@ -26,48 +26,73 @@ ENV SCCACHE_S3_NO_CREDENTIALS=false
 
 # Install system packages depending on the LINUX_VER
 RUN \
-    PKG_CUDA_VER="$(echo ${CUDA_VER} | cut -d '.' -f1,2 | tr '.' '-')"; \
     case "${LINUX_VER}" in \
       "ubuntu"*) \
         apt-get update \
         && apt-get upgrade -y \
         && apt-get install -y --no-install-recommends \
-          cuda-gdb-${PKG_CUDA_VER} \
-          cuda-cudart-dev-${PKG_CUDA_VER} \
-          cuda-cupti-dev-${PKG_CUDA_VER} \
           file \
           unzip \
           wget \
-        # ignore the build-essential package since it installs dependencies like gcc/g++
-        # we don't need them since we use conda compilers, so this keeps our images smaller
-        && apt-get download cuda-nvcc-${PKG_CUDA_VER} \
-        && dpkg -i --ignore-depends="build-essential" ./cuda-nvcc-*.deb \
-        && rm ./cuda-nvcc-*.deb \
-        # apt will not work correctly if it thinks it needs the build-essential dependency
-        # so we patch it out with a sed command
-        && sed -i 's/, build-essential//g' /var/lib/dpkg/status \
         && rm -rf "/var/lib/apt/lists/*"; \
         ;; \
       "centos"* | "rockylinux"*) \
         yum -y update \
         && yum -y install --setopt=install_weak_deps=False \
-          cuda-cudart-devel-${PKG_CUDA_VER} \
-          cuda-driver-devel-${PKG_CUDA_VER} \
-          cuda-gdb-${PKG_CUDA_VER} \
-          cuda-cupti-${PKG_CUDA_VER} \
           file \
           unzip \
           wget \
           which \
           yum-utils \
-        && rpm -Uvh --nodeps $(repoquery --location cuda-nvcc-${PKG_CUDA_VER}) \
         && yum clean all; \
         ;; \
       *) \
         echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1; \
         ;; \
     esac
 
+# Install CUDA packages, only for CUDA 11 (CUDA 12+ should fetch from conda)
+RUN \
+    case "${CUDA_VER}" in \
+      "11"*) \
+        PKG_CUDA_VER="$(echo ${CUDA_VER} | cut -d '.' -f1,2 | tr '.' '-')"; \
+        case "${LINUX_VER}" in \
+          "ubuntu"*) \
+            apt-get update \
```

Comment:
```suggestion
          "ubuntu"*) \
            echo "Installing CUDA Toolkit installation for CUDA ${CUDA_VER}" on Ubuntu; \
            apt-get update \
```

Reply:
I put a single message at the top for both cases, in commit [42535dd](https://github.com/rapidsai/ci-imgs/pull/77/commits/42535ddc83bfca4ea201f84fe3be2c45b6224060).